### DISCUSSION TITLE
Switch endianness for entire file

### DISCUSSION THREAD

**Discussion by Uzver123** [WerWolv/ImHex#1739] [Get Help] [answered]
How to switch endianness for entire binary currently ption is grayed out and not clickable.

![image](https://github.com/WerWolv/ImHex/assets/143356096/e0b1a536-b8ce-4918-b4a8-85a5b83af2cf)

**Comment by WerWolv** [comment, marked as answer]
Endianness is meaningless for single-byte values. That's why it's grayed out.
It's available if you choose any of the other visualizers that use more than 1 byte (like the `Hexadecimal (16 bits)` or `Decimal (32 bits)` ones)
